app/models/journaled/writer.rb

```diff
@@ -26,9 +26,11 @@ def initialize(journaled_event:)
 
   def journal!
     validate!
-    Journaled::DeliveryJob
-      .set(journaled_enqueue_opts.reverse_merge(priority: Journaled.job_priority))
-      .perform_later(**delivery_perform_args)
+    ActiveSupport::Notifications.instrument('journaled.event.enqueue', event: journaled_event, priority: job_opts[:priority]) do
```

Comment:
BONUS: this will emit an ActiveSupport::Notification that we can also hook into via StatsD.